client/my-sites/site-settings/manage-connection/disconnect-site-link.jsx

```diff
@@ -51,7 +51,13 @@ class DisconnectSiteLink extends Component {
 				<QuerySitePlans siteId={ siteId } />
 
 				<SiteToolsLink
-					href="#"
+					href={
+						isEnabled( 'manage/site-settings/disconnect-flow' ) ? (
+							'/settings/disconnect-site/' + siteSlug
+						) : (
+							'#'
```

Comment:
The stupid hash sign is still required so we get a pointer cursor on the `SiteToolsLink`. :roll_eyes: Shouldn't matter too much since we'll rip it out once this is in production anyway.

I'll probably follow up with a PR to set `cursor: pointer` on `Card` (which `SiteToolsLink` really is) if `href` or `onClick` is set.

Reply:
PR: #19272